src/main/java/com/battleship/backend/models/PlaceRequest.java

```diff
@@ -1,6 +1,6 @@
 package com.battleship.backend.models;
 
-public class PlaceRequest {
+public class PlaceRequest extends Request{
     Ship ship;
     int row;
```

Comment:
Do you need to use inheritance?  Both classes have the same methods, except getShip, but you could pass null as the ship argument.  Why not just use a Request object for both hits and places?

Reply:
Yeah i was considering that too. Will make the change.